### PR TITLE
Modify tasks

### PR DESCRIPTION
**Pull request by HopeNadyaHope**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,97 @@
...
+1 - one
+2 - two
+3 - three
+10 - one
+```
+
+**Output**
+
+```
+one - 1
```
Comment (egoist301) [inline README.md:60]: Sorry, `one - 1` or `one - 10`?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,97 @@
...
+two - 2
+three - 3
+```
+
+## 3. Function value finder
+
+Implement the program which gets `List<Integer>` and `required value` as parameter, creates Map<Integer, Integer> and
```
Comment (egoist301) [inline README.md]: Could you wrap `Map<Integer, Integer>`?

File: src/test/java/com/epam/mjc/collections/map/KeyValueSwapperTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                                4, "Thursday",
+                                5, "Friday",
+                                6, "Saturday",
+                                7, "Sunday",
+                                10, "Friday"),
+                        Map.of("Monday", 1,
+                                "Tuesday", 2,
+                                "Wednesday", 3,
+                                "Thursday", 4,
+                                "Friday", 5,
```
Comment (egoist301) [inline src/test/java/com/epam/mjc/collections/map/KeyValueSwapperTest.java:48]: Friday 5 or Friday 10?

File: README.md (hunk 1 above)
Comment (HopeNadyaHope) [reply README.md:60]: _If in the source map `Map<Integer, String>` there are the same values for different keys, in the
new `Map<String, Integer>` this value should map to a smaller key value._ 
So, 1<10 => one mapped to 1

File: src/test/java/com/epam/mjc/collections/map/KeyValueSwapperTest.java (hunk 3 above)
Comment (HopeNadyaHope) [reply src/test/java/com/epam/mjc/collections/map/KeyValueSwapperTest.java:48]: _If in the source map `Map<Integer, String>` there are the same values for different keys, in the
new `Map<String, Integer>` this value should map to a **smaller** key value._
So, 5<10 => Friday mapped to 5

Comment (egoist301) [reply src/test/java/com/epam/mjc/collections/map/KeyValueSwapperTest.java:48]: Okay, thanks for the clarifying

**Review by egoist301 — APPROVED** [review]

**Pull request merged**